### PR TITLE
Riley Floyd

### PR DESCRIPTION
**Pull request by RFloydUT3**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [x] The webpage links to another page (multi-page website): 10 points
- [ ] The company/organization name is lined up with the navigation menu at the top of the page: 10 points
- [ ] The title is centered in front of a background image: 10 points
- [x] The body images are sized and formatted according to the wire frame: 10 points
- [x] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [x] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

Good job! So close to mastery... You left out the CSS reset file and the top section is just an image, no title or actual coded button. Fix these things and I believe you are approaching mastery.

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [x] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [x] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [x] Properly commented code: 10 points
- [x] Properly indented code: 10 points

Specific Comments:

**Outstanding. Just remove that added padding and margin to your "container-fluid" classes and this design will look perfecto!**

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [x] Responive: 20 points
- [x] Evidence of Effective Use of Design Principles: 20 points
- [x] Links to Resume: 20 points
- [x] Links to at least 3 Projects: 20 points
- [x] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 02week/nasa/css/style.css [hunk 1]
```diff
@@ -13,17 +13,112 @@ background-size:cover;
...
+
+
+.office h1 {
+  font-size:20px;
+  padding-left:10px;
+  font-weight:bold;
+
+}
+
+.office a {
```
Comment (demarcogleason) [inline 02week/nasa/css/style.css:59]: Excellent job using that Pseudo selector!

File: 03week/blog/index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
     <section class="column main-column">
```
Comment (KeenanGoldis) [inline 03week/blog/index.html]: nice double class

File: 03week/newspaper/css/style.css [hunk 3]
```diff
@@ -1,163 +1,145 @@
...
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
-  min-width: 300px;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  text-rendering: optimizeLegibility;
+h1 {
+  margin: auto;
+  font-size:48px;
+  text-align:center;
```
Comment (KeenanGoldis) [inline 03week/newspaper/css/style.css]: Align:center is a good position!

File: 03week/newspaper/css/style.css [hunk 4]
```diff
@@ -1,163 +1,145 @@
...
-
-div, h2, p, figure {
-  margin: 0;
-  padding: 0;
+/*nav bar section*/
+ul {
+  display:flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
```
Comment (KeenanGoldis) [inline 03week/newspaper/css/style.css:16]: Can you just erase this, or is it good to state that it's zero?

File: testexample/css/style.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+}
+.flex-grid {
+  display: flex;
+  justify-content:center;
+}
+/*.flex-grid .col {
+  flex: 1;
+
+}*/
+.flex-grid-thirds {
```
Comment (KeenanGoldis) [inline testexample/css/style.css]: Nice flex-grid!

**Review by ktee1232 — COMMENTED** [review]
Hey Riley,

I'm not too concerned about the checkpoint, seems like you just need more time. Keep at it!

Kai

**Pull request closed without merging**